src/Controls/samples/Controls.Sample.Sandbox/MapPage.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Maps;
+using Microsoft.Maui.Devices.Sensors;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Maps;
+
+namespace Maui.Controls.Sample
+{
+
+	public class MapElementsPage : ContentPage
+	{
+		public MapElementsPage()
+		{
+			var grid = new Grid();
+			grid.RowDefinitions.Add(new RowDefinition());
+
+			var myhouse = new Location(47.6368678, -122.137305);
+
+			//var center = new 
+			var map = new Microsoft.Maui.Controls.Maps.Map(new MapSpan(myhouse, 0.1, 0.1));
+			grid.Children.Add(map);
+
+			// add the polygon to the map's MapElements collection
+			map.MapElements.Add(AddPolygon());
+
+			// add the polyline to the map's MapElements collection
+			map.MapElements.Add(AddPolyline());
+			//UpdatePolyLine(polyline);
```

Comment:
Intentionally left commented?

Reply:
yeah for testing like add just property, or if you want to check if the update of properties of the PolyLine works